src/domain/app/appWideNotification/AppWideNotification.tsx

```diff
@@ -0,0 +1,64 @@
+import { Notification } from "hds-react";
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+
+const IS_OPEN_KEY = "ulkoliikunta:isAppWideNotificationOpen";
+
+function getInitialValue(
+  sessionStorageValue: string | null,
+  isNotificationEnabled: boolean | null
+) {
+  if (sessionStorageValue) {
+    return sessionStorage.getItem(IS_OPEN_KEY) === "true";
+  }
+
+  return isNotificationEnabled === true;
+}
+
+export function AppWideNotification() {
+  const notificationContentTranslations: Record<string, string | undefined> = {
+    fi: process.env.REACT_APP_SITE_WIDE_NOTIFICATION_FI,
+    sv: process.env.REACT_APP_SITE_WIDE_NOTIFICATION_SV,
+    en: process.env.REACT_APP_SITE_WIDE_NOTIFICATION_EN,
+  };
+  const isNotificationEnabledEnvVariable =
+    process.env.REACT_APP_SITE_WIDE_NOTIFICATION_ENABLED;
+  const isNotificationEnabled = isNotificationEnabledEnvVariable
+    ? Boolean(JSON.parse(isNotificationEnabledEnvVariable))
+    : null;
+
+  const [isOpen, setOpen] = useState(
+    getInitialValue(sessionStorage.getItem(IS_OPEN_KEY), isNotificationEnabled)
```

Comment:
Could be callback so this wouldnt be called on every render

Reply:
Didn't even pass my mind 👍